README.md

```diff
@@ -157,3 +157,812 @@ SLINGR is a low-code rapid application development platform that accelerates dev
+            The url to which this endpoint will send the request. This is the exact service to which the http request will be made. <br>
+            Possible values are: <br>
+            <i><strong>/addresses/validate<br>/shipments<br>/shipments/{shipmentId}/tags/{tagName}<br>/tags/{tagName}<br>/warehouses<br>/labels<br>/labels/rates/{rateId}<br>/labels/shipment/{shipmentId}<br>/labels/{label_id}/return<br>/tracking/start<br>/tracking/stop<br>/batches<br>/batches/{batchId}/add<br>/batches/{batchId}/remove<br>/batches/{batchId}/process/labels<br>/rates<br>/rates/estimate<br>/rates/bulk<br>/connections/carriers/fedex<br>/connections/carriers/ups<br>/connections/carriers/stamps_com<br>/connections/carriers/endicia<br>/shipments<br>/shipments/{shipmentId}<br>/packages/{packageId}<br>/manifests/{manifestId}<br>/manifests<br>/tags<br>/warehouses/{warehouseId}<br>/warehouses<br>/labels<br>/batches/{batchId}<br>/batches<br>/batches/external_batch_id/{externalBatchId}<br>/batches/{batchId}/errors<br>/shipments/{shipmentId}/rates<br>/connections/carriers/fedex/{fedexId}/settings<br>/connections/carriers/ups/{upsId}/settings<br>/carriers<br>/carriers/{carrierId}<br>/carriers/{carrierId}/services<br>/carriers/{carrierId}/options<br>/carriers/{carrierId}/packages<br>/shipments/{shipmentId}/tags/{tagName}<br>/tags/{tagName}<br>/warehouses/{warehouseId}<br>/batches/{batchId}<br>/connections/carriers/fedex/{fedexId}<br>/connections/carriers/ups/{upsId}<br>/connections/carriers/ups/{upsId}<br>/connections/carriers/ups/{upsId}<br>/connections/carriers/stamps_com/{stampsId}<br>/connections/carriers/endicia/{endiciaId}<br>/addresses/recognize<br>/shipments/{shipmentId}<br>/packages/{packageId}<br>/tags/{tagName}/{newTagName}<br>/warehouses/{warehouseId}<br>/labels/{labelId}/void<br>/connections/carriers/fedex/{fedexId}/settings<br>/connections/carriers/ups/{upsId}/settings<br></strong></i>
+        </td>
+    </tr>
+    <tr>
+        <td>Headers</td>
+        <td>keyValue</td>
+        <td>no</td>
+        <td> - </td>
+        <td>Always</td>
+        <td>
+            Used when you want to have a custom http header for the request.
+        </td>
+    </tr>
+    <tr>
+        <td>Query Params</td>
+        <td>keyValue</td>
+        <td>no</td>
+        <td> - </td>
+        <td>Always</td>
+        <td>
+            Used when you want to have a custom query params for the http call.
+        </td>
+    </tr>
+    <tr>
+        <td>Body</td>
+        <td>json</td>
+        <td>no</td>
+        <td> - </td>
+        <td>Always</td>
+        <td>
+            A payload of data can be sent to the server in the body of the request.
+        </td>
+    </tr>
+    <tr>
+        <td>Event</td>
+        <td>dropDown</td>
+        <td>no</td>
+        <td> - </td>
+        <td>Always</td>
+        <td>
+            Used to define event after the call. <br>
+            Possible values are: <br>
+            File Downloaded, Callback
+        </td>
+    </tr>
+    <tr>
```

Comment:
We have this issue in helper generator, we do not have anymore callbacks in generic steps